ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -109,6 +127,15 @@ public class BottomSheetController: UIViewController {
         updateResizingHandleViewAccessibility()
     }
 
+    private lazy var dimmingView: UIView = {
```

Comment:
please use the existing dimming view of fluent sdk

Reply:
oops, done